### PR TITLE
Add security context for wait-for-mongodb

### LINKED ISSUE #331: Cannot deploy litmus chaoscenter cause of missing securityContext
Hello,

We are trying to deploy litmus chaoscenter but the deployment is blocked because there is no securityContext on the "wait-for-mongodb" initContainer used by the server and auth-server.

I tested with the following securityContext and the initContainer was sucessful 
```yaml
securityContext:
  allowPrivilegeEscalation: false
  readOnlyRootFilesystem: true
  runAsNonRoot: true
  runAsUser: 101
```

### PR DESCRIPTION
**Pull request by Calvinaud**
- Add securityContext in values.yaml
- Using this securityContext in auth-server-deployment.yaml and server-deployment.yaml
- Bumping Chart version to 2.16.0 (I hesitate between bumping to 2.16.0 or 2.15.12 cause not sure if this change can be considered a bug fix)
- Regenerating docs with helm-docs

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #331

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by Calvinaud** [comment]
I was not sure if this change needed to be considered as a bug fix (and bump to 2.15.11) or a added feature (and bump to 2.16.0). I can change if needed.

**Comment by Calvinaud** [comment]
Didn't see the PR, https://github.com/litmuschaos/litmus-helm/pull/212. Should I close this one in favor of the previous one ?

**Comment by Jasstkn** [comment]
Let me take a look. Will reply by EOD, thanks!

File: charts/litmus/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "2.14.0"
 description: A Helm chart to install ChaosCenter
 name: litmus
-version: 2.15.10
```
Comment (Jasstkn) [inline charts/litmus/Chart.yaml:5]: we bump only patch versions, more about this decision is [here](https://github.com/litmuschaos/litmus-helm/blob/master/CONTRIBUTING.md#versioning)

File: charts/litmus/values.yaml [hunk 2]
```diff
@@ -168,6 +168,11 @@ portal:
         repository: curl
         tag: 2.14.0
         pullPolicy: "Always"
+      securityContext:
```
Comment (Jasstkn) [inline charts/litmus/values.yaml:171]: I would prefer to have the default as empty value and let user to define it. If you want, you can leave the suggested values commented.

**Review by Jasstkn — CHANGES_REQUESTED** [review]

**Comment by Jasstkn** [comment]
> Let me take a look. Will reply by EOD, thanks!

Mentioned PR wasn't updated on requested, closed it in favor of yours. Please, let me know if you have any questions about the comments that I've left.

File: charts/litmus/values.yaml (hunk 2 above)
Comment (Calvinaud) [reply charts/litmus/values.yaml]: I would think to be put more security by default and let people remove it if needed. Updated the PR so securityContext is empty in default.

File: charts/litmus/Chart.yaml (hunk 1 above)
Comment (Calvinaud) [reply charts/litmus/Chart.yaml:5]: Switch to bumping to 2.15.11. And sorry for my last PR I bumped the minor on litmus-agent.

**Review by Jasstkn — APPROVED** [review]
LGTM 🚀 Thank you for your contribution!

@imrajdas @ispeakc0de can you take a look here?

**Comment by Jasstkn** [comment]
@Calvinaud gonna be released in a few moments!

**Pull request merged**